openfisca_extension_template/variables/local_benefit.py

```diff
@@ -19,20 +23,21 @@ class local_town_child_allowance(Variable):
     definition_period = MONTH
     label = "Local benefit: a fixed amount by child each month"
 
-    def formula(famille, period, parameters):
+    def formula(family: t.Population, period: t.Instant, params: t.Params) -> t.Array:
```

Comment:
To keep template in lie with what is done in country packages I am aware of:
- I am not sure that typing is necessary here.
- I would keep parameters instead of `param`.